starlark/value.go

```diff
@@ -131,15 +131,36 @@ type Comparable interface {
 	CompareSameType(op syntax.Token, y Value, depth int) (bool, error)
 }
 
+// Alternate, simplified comparison interface
+type ThreeWayComparable interface {
+	Value
+	// ThreeWayCompareSameType compares one value to another of the same orderable
```

Comment:
// Cmp compares two values x and y of the same totally ordered type.
// It returns negative if x < y, positive if x > y, and zero if the values are equal.

(I don't think we should specify that the values are exactly +/- 1.)